text/get_word_distance.ts

```diff
@@ -0,0 +1,12 @@
+import { levenshteinDistance } from "./levenshtein_distance.ts";
+
+/**
+ * Compare two strings by similarity.
+
+ * Currently exports {@linkcode levenshteinDistance}.
+ * Metric used may be swapped / improved in the future.
+ *
+ * @returns The difference in distance. Lower number means closer match.
+ * 0 means numbers match.
+ */
+export const getWordDistance = levenshteinDistance;
```

Comment:
This export feels redundant to me.

Reply:
As mentioned in the PR description:
> Also adds an export for getWordDistance, which was previously defined in 2 different files, so that Deno's "recommended" metric at the time can accessed by the user. Literally just exports levenshteinDistance. If this is not desired, I can undo this.

Without this export, the user has no way to get the current "recommended" Metric. Again, I can undo this if desired, and put it back into the two different files it came from.

Also, I re-exported the import instead of just using `export from` so that I could change the documentation.

Reply:
I agree with Yoshiya—we don't need this exported. The default algorithm can be obtained by reading the documentation for the new `distanceFn` option.